bytepit-ui/src/components/Navbar.tsx

```diff
@@ -0,0 +1,49 @@
+import DefaultUserImg from '../assets/default_user.png'
+import { useState } from 'react'
+
+const Navbar = () => {
+    const [isLoggedIn, setIsLoggedIn] = useState(false)
+    const [openState, setOpenState] = useState(false)
+
+    return (
+        <nav className="bg-gray-300 border-b-2 border-gray-400 w-full p-4
+            flex md:flex-row justify-between items-center h-20">
+            <div className="bg-gray-30 font-bold text-3xl max-md:flex max-md:items-center max-md:justify-between max-md:w-full">
+                BytePit
+                <svg onClick={() => setOpenState(!openState)} className="md:hidden svg-icon w-7 h-7" viewBox="0 0 20 20">
+                    {!openState ? 
+                        <path fill="black" d="M3.314,4.8h13.372c0.41,0,0.743-0.333,0.743-0.743c0-0.41-0.333-0.743-0.743-0.743H3.314 c-0.41,0-0.743,0.333-0.743,0.743C2.571,4.467,2.904,4.8,3.314,4.8z M16.686,15.2H3.314c-0.41,0-0.743,0.333-0.743,0.743 s0.333,0.743,0.743,0.743h13.372c0.41,0,0.743-0.333,0.743-0.743S17.096,15.2,16.686,15.2z M16.686,9.257H3.314 c-0.41,0-0.743,0.333-0.743,0.743s0.333,0.743,0.743,0.743h13.372c0.41,0,0.743-0.333,0.743-0.743S17.096,9.257,16.686,9.257z"></path>
```

Comment:
what is the purpose of this tag? please move it outside of this component as it is too messy. you can move to some other directory and import it from there

Reply:
It can be made similar to LF where we had one file for all SVGs we used.

Reply:
yes move it elsewhere